pipelines.yml

```diff
@@ -0,0 +1,173 @@
+# Set common JFrog integration and GitHub input source
+commonConfiguration: &COMMON_CONFIGURATION
+  integrations:
+    - name: jfrog
+  inputResources:
+    - name: jfrogCliGit
+
+# Create artifactory access token and save it in './rtToken'
+createRtAccessToken: &CREATE_RT_ACCESS_TOKEN ./build/build.sh && ./jfrog rt atc $int_jfrog_rt_user --url=$int_jfrog_rt_url --user=$int_jfrog_rt_user --password=$int_jfrog_rt_password --grant-admin --expiry 1800 | jq '.access_token' -r > rtToken
+
+# Update commit status in GitHub
+updateCommitStatus: &UPDATE_COMMIT_STATUS
+  onStart:
+    - update_commit_status jfrogCliGit --context "$step_name"
+  onComplete:
+    - update_commit_status jfrogCliGit --context "$step_name"
+
+resources:
+  - name: jfrogCliGit
+    type: GitRepo
+    configuration:
+      path: jfrog/jfrog-cli
+      gitProvider: github
+      buildOn:
+        pullRequestCreate: true
+
+pipelines:
+  # Global configuration
+  - name: test_cli
+    configuration:
+      runtime:
+        type: image
+        image:
+          custom:
+            name: releases-docker.jfrog.io/jfrog-ecosystem-integration-env
+            tag: 1.0.0
+      environmentVariables:
+        readOnly:
+          GOPROXY: https://gocenter.io
+          JFROG_CLI_OFFER_CONFIG: "false"
```

Comment:
I think that this env var is no longer required, because when the CI env var is set true, all interactive prompts are disabled anyway.
I think we should add a task for deprecating JFROG_CLI_OFFER_CONFIG.